src/components/editor/MonacoEditor.vue

```diff
@@ -110,7 +111,17 @@ async function createEditor() {
 
   editor = monaco.editor.create(container.value, { value, language });
 
-  editor.onDidChangeModelContent(updateFile);
+  editor.onDidChangeModelContent(() => {
```

Comment:
Maybe add a comment on why we are doing this ?

Reply:
changelog

Reply:
Externalize function inside `onDidChangeModelContent` and add documentation on it.

Reply:
done